README.md

```diff
@@ -1,5 +1,9 @@
-# Installation
+# cusdeb-single-node 
+Script for making easier local deploying and debugging of CusDeb project.
```

Comment:
The project is a set of scripts which is intended for simplifying deployment of CusDeb for developing purposes.